scripts/codex_runner.py

```
@@ -0,0 +1,523 @@
+        "--skip-git-repo-check",
+        "--color", "never",
+        "-o", str(last_msg),
+    ]
+    if model:
+        cmd += ["--model", model]
+
+    log(f"[codex_runner] 実行: {Path(codex_bin).name} exec --sandbox {sandbox} -> {output_path}")
+
+    env = dict(os.environ, PYTHONIOENCODING="utf-8")
+    try:
+        proc = subprocess.run(
```

Ruff (0.16.0)

[error] `subprocess` call: check for execution of untrusted input

(S603)